src/server.c

```diff
@@ -100,11 +100,26 @@ void get_d20(int fd)
     // IMPLEMENT ME! //
     ///////////////////
 
+    srand(time(NULL));
+    int r = rand() % 20;
+    char rand_num[3];
+
+    if (r < 10)
+    {
+        sprintf(rand_num, "0%i", r);
```

Comment:
jajaja always on!

Reply:
Needs a little work so the result isn’t as skewed, but gets the job done for something like this.